### DISCUSSION TITLE
Is there a way to change the order of the icons in component primeNG PanelMenu ?

### DISCUSSION THREAD

**Discussion by BySAQQ** [primefaces/community#207] [PrimeNG] [answered]
I would like to do it for this Label 
![image](https://github.com/primefaces/community/assets/94871596/0ac1c05d-09d3-45c6-86f4-ad9e51313058)

I would love to have them in this order, but I can't find the class to modify it in the .scss file.
only in the inspector I can modify it as I want
![image](https://github.com/primefaces/community/assets/94871596/53f52d55-92bb-4c7e-b453-5b82cdd16481)
![image](https://github.com/primefaces/community/assets/94871596/85040fcf-1d18-4d8c-a4c8-449718cf1412)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, you can't change the positions, PanelMenu doesn't offer such a feature. However, it's quite easy to implement manually. Here is a Stackblitz example how to do it https://stackblitz.com/edit/vam8vx?file=src%2Fapp%2Fdemo%2Fpanel-menu-basic-demo.scss,src%2Fapp%2Fdemo%2Fpanel-menu-basic-demo.ts,src%2Fapp%2Fapp.module.ts

Class names may be different in your project, you can change them as you want.

**Reply by BySAQQ** [reply]
thank you very much for helping me with this!! 😎😎
